core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/cumSum.kt

```diff
@@ -212,10 +220,11 @@ public fun <T, G> GroupBy<T, G>.cumSum(
  * {@set [CumSumDocs.DATA_TYPE] [GroupBy]}
  * {@set [CumSumDocs.CUMSUM_PARAM]}
  */
+@Refine
+@Interpretable("GroupByCumSum0")
```

Comment:
If we are marking the functions for Compiler Plugin, should we add a test for this trivial case?
@koperagen

Reply:
testing what? and how?

Reply:
No, usually i simply annotate functions and later adjust if anything is wrong (one time i forgot Refine annotation)